android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -156,12 +156,33 @@ public void sendToNotificationCentre(Bundle bundle) {
                 ApplicationInfo appInfo = context.getApplicationInfo();
                 title = context.getPackageManager().getApplicationLabel(appInfo).toString();
             }
+            NotificationCompat priority;
+            if (bundle.getString("priority")) {
```

Comment:
`bundle.getString("priority")`

You could pull that into a final var since you reference it twice, if you wanted.

Reply:
Good point, will change!